.eslintrc.yml

```diff
@@ -1,2 +1,5 @@
 extends:
     - '@telefonica/eslint-config'
+overrides:
+    - files: ['*.js']
+      parser: espree
```

Comment:
use default (espree) parser instead of babel parser for js files, because we don't have a babel config in this repo, because we don't use babel, but swc